spec/services/preserved_object_handler_check_exist_spec.rb

```diff
@@ -457,23 +437,21 @@
         end
       end
 
-      it 'calls PreservedCopy.touch (but not PreservedObject.touch) if the existing record is NOT altered' do
+      it 'calls PreservedCopy.save! if the existing record is NOT altered' do
```

Comment:
how about "calls PreservedCopy.save! (but note PreservedObject.save!) if the existing version is NOT altered" ?

The original test shows that PreservedCopy timestamps are updated, but PreservedObject was unchanged.